test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -371,5 +373,11 @@ void TestFunctionCall::reset()
 
 bool TestFunctionCall::matchesExpectation() const
 {
-	return m_failure == m_call.expectations.failure && m_rawBytes == m_call.expectations.rawBytes();
```

Comment:
I would lean to just keeping this as it is. We have the other pending proposal to flag a failure if the formatted expectation (instead of the raw expectation) is different, I would prefer to have that proposal implemented instead.

Reply:
I still think this should be done in a more generic way: Only accept the formatting that the auto-formatter would produce - and make this change independent from the fixed point feature. If you have a `uint` and the expectation is `"abc"`, then it should also be an error because integers should not be formatted as strings.

Reply:
Ok, I removed `valid()` now.